src/commands/command-docs.json

```diff
@@ -18,6 +18,188 @@
         "command_tips": [
             "NONDETERMINISTIC_OUTPUT_ORDER"
         ],
+        "reply_scema": {
+            "description": "A map where each key is a command name, and each value",
+            "type": "object",
+            "patternProperties": {
+                "^.*$": {
+                    "type": "object",
+                    "properties": {
+                        "summary": {
```

Comment:
where is the reply schema that describes `reply_schema`? :exploding_head:

(btw maybe we should add `additionalProperties: False`)